tests/Makefile

```diff
@@ -1,12 +1,18 @@
 GCC=riscv64-unknown-elf-gcc
 OBJDUMP=riscv64-unknown-elf-objdump
-CFLAGS=-mcmodel=medany -std=gnu99 -O2 -fno-common -fno-builtin-printf -Wall
-LDFLAGS=-static -nostdlib -nostartfiles -lgcc
```

Comment:
I'm going to trust you on this. Out of curiosity: does leaving out these options cause GCC to find newlib automatically? I don't see newlib mentioned explicitly anywhere in these changes.

Reply:
The gcc spec files subsume these options.  By default, the RISC-V ELF toolchain uses newlib if `-nostdlib` is omitted.